app/models/pagination.rb

```diff
@@ -4,7 +4,7 @@ class Pagination
 
   DEFAULT_PER_PAGE = 20
   DEFAULT_PAGE = 1
-  MAX_PER_PAGE = 200
+  MAX_PER_PAGE = 1000
```

Comment:
Does this impact apply and other review pages that have pagination - (search page or open/closed application pages)?

Reply:
It could yes. However, as things currently stand, the API should be able to handle returning 1000 records comfortably. If this is a problem for the client (as it might be on Review atm), then the client should handle that.